src/types/convert.rs

```diff
@@ -121,3 +121,29 @@ impl TryFrom<InitOptions> for pipeline::Options {
         Ok(result)
     }
 }
+
+const PORT_CONVERSION_ERROR_MESSAGE: &str = "Port needs to be an unsigned, 16-bit integer or a string in the \"START:END\" format, where START and END are both unsigned, 16-bit integers.";
```

Comment:
```suggestion
const PORT_CONVERSION_ERROR_MESSAGE: &str = "Port needs to be a number between 1-65535 or a string in the \"START:END\" format, where START and END represent a range of ports.";
```

Also maybe move it inside try_from